lib/matplotlib/__init__.py

```diff
@@ -1260,6 +1260,7 @@ def test(verbosity=None, coverage=False, **kwargs):
     finally:
         if old_backend.lower() != 'agg':
             use(old_backend)
+        sys.setrecursionlimit(old_recursionlimit)
```

Comment:
Actually we don't modify the recursion limit anymore, so you could instead remove the read at the top.

Reply:
OK! I wasn't sure and this seemed like the safest bet. But will change.